solutions/swb-reference/integration-tests/support/clientSession.ts

```diff
@@ -61,7 +61,9 @@ export default class ClientSession {
       },
       function (error: AxiosError) {
         if (error.response) {
-          return Promise.reject(new HttpError(error.response.status, error.response.data));
+          const httpError = new HttpError(error.response.status, error.response.data);
+          console.error(JSON.stringify(httpError));
```

Comment:
remove this

Reply:
Was it removed?

Reply:
i actually kept it for logging purposes.